config/application.rb

```diff
@@ -44,7 +44,8 @@ class Application < Rails::Application
 
     # Configure sensitive parameters which will be filtered from the log file.
     config.filter_parameters += [:password, :password_confirmation]
-    #config.action_mailer.default_url_options = { :host => "localhost" }
+
+    config.action_mailer.default_url_options = { host: 'opensnp.org' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.